docs/devenv_rhel8.md

```diff
@@ -85,10 +85,10 @@ make srpm
 The artifacts of the build are located in the `packaging` directory.
```

Comment:
Need to fix the sentence above - the build artifacts are no longer located in `packaging` dir